test/acceptance/test/add_tests.go

```diff
@@ -260,7 +273,7 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		By("Then I should see relevant message in the console", func() {
 
 			Eventually(addCommandOutput).Should(MatchRegexp(`Checking cluster status[.?]+ (Unknown|Unmodified)`))
-			Eventually(addCommandErr).Should(ContainSubstring("WeGO not installed... exiting"))
+			Eventually(addCommandErr).Should(MatchRegexp(`WeGO.*... exiting`))
 		})
 	})
 
```

Comment:
Shouldn't we verify that wego is exiting _because_ it's not installed?

Reply:
You are right but at the moment it's not very clear what Unknown status means and the difference between Unmodified & Uknown. Could you please verify that with @jrryjcksn